.travis.yml

```diff
@@ -5,7 +5,6 @@ php:
     - 5.5
     - 5.6
     - 7.0
-    - hhvm
```

Comment:
As a temp measure as it makes it impossible to test the way some exceptions are handled.